app/helpers/projects_helper.rb

```diff
@@ -3,4 +3,23 @@ def estimated(story)
     @estimate_id = story.estimate_for(current_user)
     @estimate_id.present?
   end
+
+  def link_unless_archived(project, text, url, color: nil, method: :get, remote: false, icon: nil, data_attr: nil)
```

Comment:
maybe we can rename `color` to `extra_classes` or `classes` or `class`? it's not always used to set the color (I see `color: "edit"` and `color: "add-estimate"`)